### PR TITLE
issue #180 - ensure that dozer, jaxb, and jackson deps are added to pom

### PR DESCRIPTION
**Pull request by bfitzpat**

### CODE REVIEW

File: transformation/plugins/org.jboss.tools.fuse.transformation.editor/src/main/java/org/jboss/tools/fuse/transformation/editor/wizards/NewTransformationWizard.java [hunk 1]
```diff
@@ -170,6 +168,18 @@ public void run(IProgressMonitor monitor) throws InterruptedException, Invocatio
...
+                        // make sure the camel-dozer dependency gets in
+                        ComponentDependency dep = createDependency("org.apache.camel", 
+                                "camel-dozer", 
+                                org.fusesource.ide.camel.editor.Activator.getDefault().getCamelVersion());
+                        List<ComponentDependency> deps = new ArrayList<>();
+                        deps.add(dep);
+                        try {
+                            MavenUtils.updateMavenDependencies(deps);
+                        } catch (CoreException e) {
+                            e.printStackTrace();
```
Comment (lhein) [inline transformation/plugins/org.jboss.tools.fuse.transformation.editor/src/main/java/org/jboss/tools/fuse/transformation/editor/wizards/NewTransformationWizard.java]: better user activator log method

**Comment by bfitzpat** [comment]
Hold off on this. Found a logic issue with what I did...

File: transformation/plugins/org.jboss.tools.fuse.transformation.editor/src/main/java/org/jboss/tools/fuse/transformation/editor/wizards/NewTransformationWizard.java [hunk 2]
```diff
@@ -502,8 +519,36 @@ private void addDataFormat(CamelContextFactoryBean context, DataFormatDefinition
...
+                    "camel-jaxb", 
+                    org.fusesource.ide.camel.editor.Activator.getDefault().getCamelVersion());
+        }
+        if (dep != null) {
+            List<ComponentDependency> deps = new ArrayList<>();
+            deps.add(dep);
+            try {
+                MavenUtils.updateMavenDependencies(deps);
+            } catch (CoreException e) {
+                e.printStackTrace();
```
Comment (lhein) [inline transformation/plugins/org.jboss.tools.fuse.transformation.editor/src/main/java/org/jboss/tools/fuse/transformation/editor/wizards/NewTransformationWizard.java]: Activator log method

Comment (bfitzpat) [inline transformation/plugins/org.jboss.tools.fuse.transformation.editor/src/main/java/org/jboss/tools/fuse/transformation/editor/wizards/NewTransformationWizard.java]: Ah. Ok. I'll add that. One sec.

**Comment by bfitzpat** [comment]
Updated to log the errors and changed it so we add maven dependencies in both the standalone and in-camel-editor cases.

**Comment by kcbabo** [comment]
pushed

**Pull request closed without merging**